docs/changelog/v5.md

```diff
@@ -4,14 +4,18 @@ slug: latest
 sidebar_label: Latest (v5)
 sidebar_position: 1
 ---
+# 5.2.2
+- tx
+  - Fix `BooleanWitnessCondition` serialization.
+  - Fix `CalledByContractCondition` missing `0x` for `hash` in `toJson`.
+  - Add test cases for WitnessCondition serialization.
```

Comment:
I would leave out any internal comments in the changelog.

Reply:
you mean the `add test cases`?

Reply:
yeah there is no need to mention test cases in the changelog imo, the users are more likely to be interested in docs/api changes.